src/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -263,5 +313,26 @@ const getAvailablePaymentMethods = (
     )
   }
 
+  // when available, unshift SEPA since it's the first option we want to offer
```

Comment:
I think we can add more context to this, otherwise, it will be confusing to have both SEPA and ACH as the preferred options. We can say, "...we want to offer to US." and "...we want to offer for EU.".
What do you think?